server/quotad.py

```diff
@@ -107,9 +107,13 @@ def start():
     """
     Start Quota Manager
     """
+    first_time = True
     while True:
         crawl()
         time.sleep(1)
+        if not first_time:
+            print("Successfully started quotad process")
```

Comment:
Any specific reason for not adding the print statement before while loop?

Reply:
I thought I should write a log after the first crawl(), so that we are very sure things are ready? Happy to change.